medical-hub/src/component/Labtest/labtest.jsx

```diff
@@ -1,521 +0,0 @@
-}
-#main>div>img{
-    width: 100%;
-}
-.package{
-    text-align: left;
-    margin-top: 15px;
-}
-.package>button{
-    padding: 10px 36px;
-    background-color: rgb(255,111,97);
-    border: none;
-    color: white;
-    font-weight: bold;
-    border-radius: 5px;
-}
-#popular{
-    padding-top: 30px;
-    padding-left: 52px;
-    margin: auto;
-    width: 100%;
-}
-#popular>h3{
-    text-align: left;
-}
-#popular1{
-    display: flex;
-    width: 80%;
-    margin-top: 20px;
-}
-@media only screen and (max-width: 1160px) { 
-    #popular1{
-        flex-wrap: wrap;
-}
-#FeaturedLabs>div:nth-child(2){
-    flex-wrap: wrap;
-}
-}
-.popular2{
-width: 80%;
-border: 1px solid rgb(232,234,236);
-padding: 15px 0px 20px 30px;
-}
-.popular2>div:nth-child(1){
-display: flex;
-}
-.popular2>div:nth-child(1)>div>h5{
-margin: 10px;
-}
-.popular2>div:nth-child(2){
-display: grid;
-grid-template-columns: repeat(2,1fr);
-/* gap: 60px; */
-justify-content: space-between;
-}
-.popular2>div:nth-child(2)>div{
-padding: 8px 0px 0px 8px;
-}
-.popular2>div:nth-child(2)>div>div{
-display: flex;
-gap: 10px;
-margin-top: 10px;
-}
-.popular2>div:nth-child(2)>div>div>div:nth-child(1){
-    margin-top: 2px;
-}
-#join{
-    margin: auto;
-    width: 90%;
-    margin-top: 40px;
-}
-#join + h2{
-    font-size: 20px;
-    font-weight: 700;
-    color: #212121;
-    margin-top: 40px;
-    margin-left: 70px;
-    text-align: left;
-}
-#popularPackage{
-display: flex;
-margin-top: 5px;
-margin-left: 70px;
-}
-#popularPackage>div{
-    margin: 4px 6px 4px 0;
-    font-size: 12px;
-    padding: 8px;
-    border-radius: 20px;
-    display: inline-flex;
-    border: 1px solid #616161;
-    color: #212121;
-    cursor: pointer;
-}
-#popularPackage>div:hover{
-    background-color: #616161;
-    color: white;
-}
-#FeaturedLabs{
-background-color: rgb(251, 251, 251);
-padding-top: 30px;
-margin-top: 30px;
-}
-#FeaturedLabs>div:nth-child(1){
-    width: 90%;
-margin: auto;
-    font-size: 20px;
-    line-height: 1;
-    margin-bottom: 30px;
-    opacity: .87;
-    font-size: 16px;
-    font-weight: 700;
-    color: #212121;
-    text-align: left;
-}
-#FeaturedLabs>div:nth-child(2){
-    display: flex;
-    justify-content: space-between;
-    padding: 0px 60px 60px 60px;
-}
-#FeaturedLabs>div:nth-child(2)>div>div:nth-child(1){
-    border: 1px solid rgb(232,234,236);
-    border-radius: 5px;
-    padding: 15px;
-}
-#FeaturedLabs img{
-    height: 60px;
-    width: 150px;
-    
-}
-#rating{
-    display: flex;
-    justify-content: center;
-    gap: 20px;
-    height: 100%;
-}
-#rating>div:nth-child(1){
-    height: 25px;
-    background-color: #1aaa2a;
-    font-family: Roboto,Helvetica Neue,Helvetica,Arial,sans-serif;
-    font-weight: 500;
-    display: inline-block;
-    color: #fff;
-    border-radius: 2px;
-    min-width: 34px;
-    padding: 2px;
-    display: inline-flex;
-    justify-content: space-evenly;
-}
-.FeaturedPara{
-    margin-top: 20px;
-    font-size: 11px;
```

Comment:
always give font-size, margin, padding etc. values in the multiple of 4. ex: 4px, 8px, 12px, 16px,